build.gradle

```diff
@@ -27,9 +27,37 @@ buildscript {
 repositories {
     mavenLocal()
     jcenter()
+    maven {
+        credentials {
+            username smartThingsArtifactoryUserName
+            password smartThingsArtifactoryPassword
+        }
+        url "http://artifactory.smartthings.com/libs-release-local"
```

Comment:
I know how stuff like that tends to slip, but I don't want to shave the yak too far on this one. We currently aren't even *compiling* this code before we try to integrate it.